pkgs/development/tools/jq/default.nix

```diff
@@ -1,33 +1,35 @@
-{stdenv, fetchurl, oniguruma}:
-let
-  s = # Generated upstream information
-  rec {
-    baseName="jq";
-    version="1.5";
-    name="${baseName}-${version}";
-    url="https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz";
-    sha256="0g29kyz4ykasdcrb0zmbrp2jqs9kv1wz9swx849i2d1ncknbzln4";
+{ stdenv, lib, fetchFromGitHub, autoreconfHook,  oniguruma }:
+
+stdenv.mkDerivation rec {
+  name = "jq-${version}";
+  version = "something";
+
+  # Fix for CVE-2015-8863 is in master,
+  # but no release has been made since then (latest release was last year)
```

Comment:
The latest jq release was last August, and there have been numerous changes in between. I've never been a fan of "backporting culture" and I wouldn't feel comfortable blindly extracting a patch and applying it without studying the source in depth to see more precisely what has changed.